tests/Tests.elm

```diff
@@ -1,6 +1,7 @@
 module Main exposing (..)
 
-import ElmTest exposing (..)
+import Test exposing (..)
+import Test.Runner.Html
```

Comment:
For now I went with the html test runner. We might consider the [node test runner](https://github.com/rtfeldman/node-test-runner), but then you have to use `npm install` and I'd rather not have to do that.